pkg/controllers/consolidation/metrics.go

```diff
@@ -48,7 +47,18 @@ var consolidationReplacementNodeInitializedHistogram = prometheus.NewHistogram(
 		Help:      "Amount of time required for a replacement node to become initialized.",
 		Buckets:   metrics.DurationBuckets(),
 	})
+var consolidationActionsPerformedCounter = prometheus.NewCounterVec(
+	prometheus.CounterOpts{
+		Namespace: metrics.Namespace,
+		Subsystem: "consolidation",
+		Name:      "actions_performed",
+		Help:      "Number of consolidation actions performed. Labeled by action.",
+	},
+	[]string{"action"},
+)
 
+// DEPRECATED: Use shared metric nodes_created instead of this metric to get number of nodes created by Karpenter
+// and the reason that nodes were created
 var consolidationNodesCreatedCounter = prometheus.NewCounter(
```

Comment:
Any reason to leave this around instead of just removing them? It's not clear that users will discover that these metrics are deprecated. Minimally, we could update the "Help" section with "[Deprecated]" or something.

Reply:
I was more concerned with breaking users that had dependency on these metrics. If we are ok with doing that, then we can just remove them.

Reply:
If we bump minor version and add a breaking conventional commit, we should be good.